frontend/vue/components/UserAccount/PrivacySection.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <section class="privacy-section">
+    <h3 class="privacy-section__title">
+      Privacy
+    </h3>
+    <h4 class="privacy-section__subtitle privacy-section__subtitle_general">
+      IBM Privacy Statement
+    </h4>
+    <AppLink class="privacy-section__policy-link" url="https://www.ibm.com/privacy">
+      https://www.ibm.com/privacy
+    </AppLink>
+    <h4 class="privacy-section__subtitle privacy-section__subtitle_quantum">
+      IBM Quantum End User Agreement
+    </h4>
+    <!-- eslint-disable-next-line vue/no-v-html -->
+    <div class="privacy-section__policy-text" v-html="privacyPolicyHTML" />
+  </section>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue-demi'
+import MarkdownIt from 'markdown-it'
+import axios, { AxiosResponse } from 'axios'
+import AppLink from '../common/AppLink.vue'
+
+export default defineComponent({
+  name: 'PrivacySection',
+  components: {
+    AppLink
+  },
+  data () {
+    return {
+      options: {
+        html: true,
+        xhtmlOut: true,
+        breaks: true,
+        linkify: true
+      },
+      privacyPolicyMd: 'Loading...'
+    }
+  },
+  computed: {
+    markdownItInstance (): MarkdownIt { return MarkdownIt(this.options) },
+    privacyPolicyHTML (): string { return this.markdownItInstance.render(this.privacyPolicyMd) }
+  },
+  mounted () {
+    axios.get('/markdown/privacy-policy.md').then(
+      (response: AxiosResponse<string>) => {
+        this.privacyPolicyMd = response.data
+      },
+      () => {
+        this.privacyPolicyMd = 'Error loading privacy policy text.'
+      }
+    )
```

Comment:
rather than having to make additional request to get/load the policy text. could we just import the markdown file into the component?

for example: https://dev.to/praveenpuglia/how-to-import-markdown-files-in-vue-cli-typescript-projects-12ca

(maybe you already looked at that approach and decided against it)